src/callbacks_step/glm_speed_dg.jl

```diff
@@ -8,3 +8,13 @@ function calc_dt_for_cleaning_speed(cfl::Real, mesh,
   #      as well if that callback changes.
   return cfl * 2 / (nnodes(dg) * max_scaled_speed_for_c_h)
 end
+
+function calc_dt_for_cleaning_speed(cfl::Real, mesh,
+  equations::AbstractIdealGlmMhdMulticomponentEquations, dg::DG, cache)
```

Comment:
Here is a similar situation wrt to the types

Reply:
done in ffdede8